src/rust/engine/process_execution/src/remote/streaming_tests.rs

```diff
@@ -29,6 +29,8 @@ use crate::remote_tests::{
 };
 use crate::{CommandRunner, Context, MultiPlatformProcess, Platform, Process};
 
+const OVERALL_DEADLINE_SECS: u64 = 10 * 60;
```

Comment:
In case you change the constructor to take a `Duration`, note that `Duration` can be used in a const position (ie, here).

Reply:
Done.